resteasy-wadl-wildfly/README.md

```diff
@@ -0,0 +1,116 @@
+# An example showing how to deploy resteasy-wadl based project to Wildfly
+
+## USAGE
+
+To deploy this sample project into Wildfly, firstly you need to upgrade your Wildfly RESTEasy module to use the same 
+version with the RESTEasy version used in this example. To achieve the goal, please refer to this section of the RESTEasy
+document:
+
+* [Upgrading RESTEasy within WildFly](https://docs.jboss.org/resteasy/docs/4.5.8.Final/userguide/html_single/index.html#upgrading-wildfly)
+
+After upgrading the Wildfly RESTEasy module to same version of the RESTEasy dependency of this project, the next step is
+to package this project with the following Maven command:
```

Comment:
I'm wondering if running on at least the same version of RESTEasy pulled by the examples is not always a pre-requisite and hence this would have to be removed from here. WDYT?

Reply:
Let me rephrase the test here.